go/vt/vtgate/planbuilder/operators/table.go

```diff
@@ -79,6 +79,13 @@ func (to *Table) AddCol(col *sqlparser.ColName) {
 	to.Columns = append(to.Columns, col)
 }
 
+func (to *Table) TablesUsed() []string {
+	if sqlparser.SystemSchema(to.QTable.Table.Qualifier.String()) {
```

Comment:
I'm just preserving logic that was already in the code, but am not sure why we do this?